dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@dbeatty/lift-shift-array-macros#egg=dbt-core&subdirectory=core
```

Comment:
remove this change

Reply:
See explanation [here](https://github.com/dbt-labs/dbt-bigquery/pull/308#issuecomment-1254396335):

> This is ready for review, and it depends on [dbt-labs/dbt-core#5823](https://github.com/dbt-labs/dbt-core/pull/5823).
> 
> After [dbt-labs/dbt-core#5823](https://github.com/dbt-labs/dbt-core/pull/5823) is approved and merged, I will revert the changes to `dev-requirements.txt` before merging this PR.